devmgmtui/src/components/user/UpdatePassword.js

```diff
@@ -20,22 +22,41 @@ class UpdatePassword extends Component {
         super(props);
 
         this.state = {
-            password: ""
+            password: "",
+            user: props.match.params.username
         }
     }
 
     componentWillMount() {
       document.title = "Update Password";  
     }
 
+    handleUserChange(e) {
+        this.setState({
+            user: e.target.value
+        })
+    }
+
     handlePasswordChange(e) {
         this.setState({
             password: e.target.value
         })
     }
 
     handleSubmit() {
-        /* Handler code goes here */
+        console.log("reached handleSubmit");
```

Comment:
Remove this console.log()

Reply:
This is removed in the commit with ID c25b777